### PR TITLE
Add some detail to tear down doc

### PR DESCRIPTION
**Pull request by willingc**
Addresses #86 

Refers to Zero to K documentation for shrinking a cluster and deleting a cluster.

### CODE REVIEW

File: doc/turn-off.rst [hunk 1]
```diff
@@ -1,9 +1,26 @@
...
 to remember to delete each of these containers or else they will continue
 to exist (and cost money!).
 
-<<<TODO>>>
+Contracting the size of your cluster
+------------------------------------
+
+If you would like to shrink the size of your cluster, refer to the
+`Expanding and contracting the size of your cluster <https://zero-to-jupyterhub.readthedocs.io/en/latest/extending-jupyterhub.html#expanding-and-contracting-the-size-of-your-cluster>`_
+section of the Zero to Jupyter documentation. Resizing the cluster to zero nodes
```
Comment (choldgraf) [inline doc/turn-off.rst]: This sounds reasonable to me, but I'd like to hear @yuvipanda 's opinion on this too...we're not unintentionally asking people do to something they'd regret later by doing this, are we?

File: doc/turn-off.rst [hunk 2]
```diff
@@ -1,9 +1,26 @@
...
+could be used if you wish to temporarily reduce the cluster (and save costs)
+without deleting the cluster.
+
+Deleting the cluster
+--------------------
+
+To delete a Binder cluster, follow the instructions in the
+`Turning Off JupyterHub and Computational Resources <https://zero-to-jupyterhub.readthedocs.io/en/latest/turn-off.html>`_
+section of the Zero to Jupyter documentation.
```
Comment (choldgraf) [inline doc/turn-off.rst]: ~Jupyter~ JupyterHub. Also we might as well hyperlink those words too :-)

File: doc/turn-off.rst [hunk 3]
```diff
@@ -1,9 +1,26 @@
...
+
+Double check your cloud provider account to make sure all resources have been
```
Comment (choldgraf) [inline doc/turn-off.rst]: maybe make this a note, and add a sentence that semi-frequently resources don't get deleted for whatever reason so this is not a small-probability kind of statement :-)

**Comment by choldgraf** [comment]
looks good! a couple comments then I'm +1

File: doc/turn-off.rst (hunk 1 above)
Comment (minrk) [reply doc/turn-off.rst]: Yes, resizing to zero is safe. It doesn't delete the master node, which is managed separately by the cloud provider, and doesn't delete volumes or deployments.

**Comment by choldgraf** [comment]
looks great, thanks! :-)

**Pull request merged**